scripts/ex-gwf-u1gwfgwf.py

```diff
@@ -1,4 +1,4 @@
-# ## USG-ex1 with GWF-GWF Exchange and XT3D
+# ## Nested Grid Problem, Two Domains
```

Comment:
Did we use this title already?  Should they be unique?

Reply:
It looks like `ex-gwf-u1disv` is named "Nested Grid Problem" and `ex-gwf-u1gwfgwf` is "Nested Grid Problem, Two Domains". Is that OK or should they be named differently?

Reply:
You're right.  Thought I saw it elsewhere but it was a different section.  Looks good.